firefox-ios/Shared/Strings.swift

```diff
@@ -2660,6 +2660,28 @@ extension String {
                     value: "Reset Site Settings",
                     comment: "Button to reset specific zoom site levels back to default")
             }
+            public struct NavigationToolbar {
+                public static let SectionHeader = MZLocalizedString(
+                    key: "Settings.Appearance.NavigationToolbar.SectionHeader.v143",
```

Comment:
we're pass string freeze for v143. I believe we're in v145. Maybe double check with l10n team if this okay?

Reply:
I thought the string version should always be the version the string is introduced in. As this is all behind a feature flag we can target only English speaking for now until we have the translations.

Reply:
@thatswinnie The version key should be the version that string is riding the train in. That means it should be `"Settings.Appearance.NavigationToolbar.SectionHeader.v145"`. We can check https://whattrainisitnow.com/ to check what nightly version we are in.

Reply:
FYI strings updated in https://github.com/mozilla-mobile/firefox-ios/pull/29651